samples/sensor/ina219/README.rst

```diff
@@ -0,0 +1,46 @@
+.. _ina219:
+
+INA219 Bidirectional Power/Current Monitor
+##########################################
+
+Overview
+********
+
+This sample application measures shunt voltage, bus voltage, power and current
+every 2 seconds and prints them to console.
+The calibration/configuration parameters can be set in the devicetree file.
+
+References
+**********
+
+ - `INA219 sensor <https://www.ti.com/product/INA219>`
```

Comment:
This is missing a trailing underscore, 
```rst
`text <url>`_
```

Reply:
thanks I've updated it. RST has a bit of a weird syntax imo.
now the tests should run fine.